### DISCUSSION TITLE
Bound marker and path element while using ImageOverlay

### DISCUSSION THREAD

**Discussion by Akshay-sheth** [geoman-io/leaflet-geoman#1203] [Questions & Help]
First of all thanks to develop and maintain such a good leaflet plugin.

I want to bound our draw element (e.g. Rectangle) on image overlay so it can't be draw or dragged out side (Grey area) of image. How to do it? 

Reference: https://jsfiddle.net/AkshaySheth/cvk3h5xq/1/

![Screenshot_3](https://user-images.githubusercontent.com/40789066/178020172-f3191de5-d89d-436f-877a-99e0958d41aa.png)

**Comment by Falke-Design** [comment]
There is currently no way to do this with geoman. You need to implement this by your self.

There is already a Feature Request for this. You can upvote it: https://github.com/geoman-io/leaflet-geoman/issues/1000
